examples/logs/exporters/otlp_http_json_with_trace.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+use OpenTelemetry\API\Common\Log\LoggerHolder;
+use OpenTelemetry\API\Logs\LogRecord;
+use OpenTelemetry\Contrib\Otlp\LogsExporter;
+use OpenTelemetry\Contrib\Otlp\OtlpHttpTransportFactory;
+use OpenTelemetry\Contrib\Otlp\SpanExporter;
+use Opentelemetry\Proto\Logs\V1\SeverityNumber;
+use OpenTelemetry\SDK\Common\Instrumentation\InstrumentationScopeFactory;
+use OpenTelemetry\SDK\Logs\LoggerProvider;
+use OpenTelemetry\SDK\Logs\LogRecordLimitsBuilder;
+use OpenTelemetry\SDK\Logs\Processor\SimpleLogsProcessor;
+use OpenTelemetry\SDK\Trace\SpanProcessor\SimpleSpanProcessor;
+use OpenTelemetry\SDK\Trace\TracerProvider;
+use Psr\Log\LogLevel;
+
+require __DIR__ . '/../../../vendor/autoload.php';
+
+/**
+ * This example demonstrates sending logs and traces via OTLP http/json. The logs should contain the trace id
+ * and span id of the active span.
+ */
+
+echo 'Starting OTLP+json with trace example' . PHP_EOL;
+
+LoggerHolder::set(
+    new Logger('otel-php', [new StreamHandler(STDOUT, LogLevel::DEBUG)])
+);
+
+$transport = (new OtlpHttpTransportFactory())->create('http://collector:4318/v1/logs', 'application/json');
+$exporter = new LogsExporter($transport);
```

Comment:
I would suggest to change name to `logsExporter` or move this into initialization of `SimpleLogsProcessor` to be consistent with `traceExporter`